projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.ts

```diff
@@ -22,7 +24,7 @@ let nextUniqueId = 0;
   encapsulation: ViewEncapsulation.None,
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
-  imports: [ LgHeadingComponent, LgIconComponent ],
+  imports: [ LgHeadingComponent, LgIconComponent, NgClass ],
```

Comment:
Do we need to import the `NgClass` in this case?

Reply:
No, we don't, I'll remove it